draft-ietf-quic-transport.md

```diff
@@ -677,66 +669,242 @@ Version negotiation uses unprotected data. The result of the negotiation MUST
 be revalidated once the cryptographic handshake has completed (see
 {{version-validation}}).
 
+
 ## Crypto and Transport Handshake {#handshake}
 
 QUIC relies on a combined crypto and transport handshake to minimize connection
-establishment latency.  QUIC provides a dedicated stream (Stream ID 1) to be
-used for performing a combined connection and security handshake (streams are
-described in detail in {{streams}}).  The crypto handshake protocol encapsulates
-and delivers QUIC's transport handshake to the peer on the crypto stream.  The
-first QUIC packet sent by client to the server MUST carry handshake information
-as data on Stream ID 1.
+establishment latency.  QUIC allocates stream 1 for the cryptographic handshake.
+This version of QUIC uses TLS.
+
+QUIC provides this stream with reliable, ordered delivery of data.  In return,
+TLS provides QUIC with:
+
+* authenticated key exchange, where
+
+   * a server is always authenticated,
+
+   * a client is optionally authenticated,
+
+   * every connection produces distinct and unrelated keys,
+
+   * keying material is usable for packet protection for both 0-RTT and 1-RTT
+     packets, and
+
+   * 1-RTT keys have forward secrecy
+
+* authenticated values for the transport parameters of the peer (see
+  {{transport-parameters}})
+
+* authenticated confirmation of version negotiation (see {{version-validation}})
+
+* authenticated negotiation of an application protocol (TLS uses ALPN
+  {{?RFC7301}} for this purpose)
+
+* for the server, the ability to carry data that provides assurance that the
+  client can receive packets that are addressed with the transport address that
+  is claimed by the client (see {{source-address-token}})
+
```

Comment:
This is the first part of the handling we agreed to.  The cryptographic protocol carries the data (though the cryptographic protocol itself is carried by QUIC).  It does this .... well, I won't reiterate why when I have a PR that goes through and implements the full client address validation we agreed to in #275.  I encourage you to read that and not get too excited about this particular PR coming up short.